salt/kubelet/kubelet.jinja

```diff
@@ -31,6 +31,11 @@ KUBELET_ARGS="\
   {% endif -%}
 {% endif -%}
     {{ pillar['components']['kubelet']['args'] }} \
+{% if salt.caasp_cri.cri_name() == 'crio' -%}
+    --container-runtime=remote \
+    --container-runtime-endpoint=/var/run/crio/crio.sock \
```

Comment:
There are many places where we are referencing this socket. Maybe we should add it to the pillar...